lib/errorHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse } from "next";
```

Comment:
@mfts, I created this kind of helper function for handling those repeated team queries and checkings.
Am I doing okay?